book/intro.md

```diff
@@ -1,11 +1,123 @@
-# Welcome to your Jupyter Book
+# 🌍 Crafting beautiful maps with PyGMT
 
-This is a small sample book to give you a feel for how book content is
-structured.
-It shows off a few of the major file types, as well as some sample content.
-It does not go in-depth into any particular topic - check out [the Jupyter Book documentation](https://jupyterbook.org) for more information.
+## Welcome to the EGU22 PyGMT short course 🥳
 
-Check out the content pages bundled with this sample book to see more.
+This Jupyter book 📖 contains [PyGMT](https://www.pygmt.org/v0.6.1) tutorials
+for producing maps 🗺️ and doing geospatial data processing 🌐
 
-```{tableofcontents}
+````{panels}
+:column: col-3 align-items-center p-2
+:body: text-center
+
+---
+:img-top: _images/first-figure_49_0.png
+```{link-button} first-figure
+:text: Anatomy of a PyGMT figure
+:type: ref
+```
+by [Leonardo Uieda](https://orcid.org/0000-0001-6123-9515)
+
+---
+:img-top: _images/ecosystem_13_1.png
+```{link-button} ecosystem
+:text: Integration with the scientific Python ecosystem 🐍
+:type: ref
+```
+by [Max Jones](https://orcid.org/0000-0003-0180-8928)
+
+---
+:img-top: https://user-images.githubusercontent.com/23487320/168939109-e35c9d85-4b2d-4623-abd8-66ed1148cf21.png
+```{link-button} Mars_Maps
+:text: Making some Mars maps with PyGMT
+:type: ref
+```
+by [André Luiz Belém](https://orcid.org/0000-0002-8865-6180)
+
+---
+:img-top: _images/lidar_to_surface_36_0.png
+```{link-button} lidar_to_surface
+:text: LiDAR Point clouds to 3D surfaces ✨➡️🏔
+:type: ref
+```
+by [Wei Ji Leong](https://orcid.org/0000-0003-2354-1988)
+
+````
+
+Each tutorial is rendered on this website for easy viewing 👀, but they are all
+Jupyter notebooks designed to be ran interactively 💫. See the instructions
+below on how you can start running the tutorials in no time! 🚀
+
+# 🌠 Quickstart
+
+To run these notebooks in an interactive Jupyter session online,
+🖱️ click on the button below to launch on regular
+[Binder](https://mybinder.readthedocs.io/en/latest/index.html).
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/GenericMappingTools/egu22pygmt/main)
```

Comment:
Note to self, add another Binder and Google Colab option as backups in case too many people try to use this at once during EGU.

Reply:
Form to request additional resouces for MyBinder: https://github.com/jupyterhub/mybinder.org-deploy/issues/new?assignees=&labels=impact&template=request_resources.md.

Reply:
Have added a backup Pangeo Binder link in 10593904d87fd4c5ff4a5c9a1b7f85de022f9d7c. Haven't got the bandwidth to do Google Colab but maybe someone can work on it in a separate PR.